packages/fxa-auth-server/scripts/verification-reminders.js

```diff
@@ -98,15 +99,15 @@ async function run () {
           case error.ERRNO.BOUNCE_COMPLAINT:
           case error.ERRNO.BOUNCE_HARD:
           case error.ERRNO.BOUNCE_SOFT:
-            console.log(`  * ignoring deleted/bouncing account ${uid}, errno: ${errno}`);
+            console.log(isotimestamp(), `  * ignoring deleted/bouncing account ${uid}, errno: ${errno}`);
             try {
               await verificationReminders.delete(uid);
             } catch (ignore) {
             }
             break;
           default:
-            console.log(`  * failed ${uid}, errno: ${errno}`);
-            console.error(err.stack);
+            console.log(isotimestamp(), `  * failed ${uid}, errno: ${errno}`);
+            console.error(isotimestamp(), err.stack);
```

Comment:
No biggie, but maybe we don't need it on the stack trace here as the preceeding log line already has the timestamp (and the stack trace spans multiple lines anyway of course).

Reply:
I removed that `isotimestamp()` for that log line.